CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ### Breaking Changes
 
 ### Enhancements
+* Support for `MutableRealm.deleteAll()`.
+* Support for `MutableRealm.delete()`.
```

Comment:
```suggestion
* Support for `MutableRealm.delete(KClass)`.
```

Reply:
yeah, the `delete(className)` is implicitly tested through `deleteAll()` but is probably better to have explicit test cases for it as well

Reply:
Yes, we try to have explicit unit tests of all public API's. This way we make sure we always test it, even if we refactor `deleteAll()` at some point.